cmd/gf/internal/cmd/genctrl/genctrl.go

```diff
@@ -34,6 +34,7 @@ gf gen ctrl
 	CGenCtrlBriefSdkStdVersion = `use standard version prefix for generated sdk request path`
 	CGenCtrlBriefSdkNoV1       = `do not add version suffix for interface module name if version is v1`
 	CGenCtrlBriefClear         = `auto delete generated and unimplemented controller go files if api definitions are missing`
+	CGenCtrlControllerMerge    = `controller fragment file merges single file`
```

Comment:
`generate all controller files into one go file by name of api definition source go file`